docs/source/gbd2019_models/risk_exposures/low_birthweight_short_gestation/index.rst

```diff
@@ -194,14 +194,65 @@ appropriate location IDs for the model you're working on):
     (sex_id=[1,2]) since we will be initializing our population using GBD's
     population data and stratifying by sex.
 
+Correcting LBWSG exposure data pulled from GBD 2019
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. important::
+
+  **The GBD 2019 exposure data for Low Birthweight and Short Gestation is wrong!**
+
+  Namely, the prevalences of the LBWSG categories returned by ``get_draws`` do **not** add up to 1! To fix the problem, follow these steps:
+
+  1.  Drop rows of the exposure data with ``'parameter' == 'cat125'`` (these
+      are precisely the rows with ``'modelable_entity_id' == NaN``). cat125 is
+      not a modeled category but rather a residual category automatically added
+      by ``get_draws`` because the prevalences that the LBWSG modelers gave to
+      central comp did not add up to 1 in each draw.
+
```

Comment:
how large is the exposure in this residual category? is it close to negligible or is it a substantial proportion of total exposure?

Reply:
If I remember correctly from last December, I think it was at most around 10% for Kenya (haven't checked other countries yet). When I have a chance, I can dig into the data a bit more and make another PR that explains this data issue in more detail.